renku/cli/init.py

```diff
@@ -133,29 +157,18 @@
 CI_TEMPLATES = [_GITLAB_CI, _DOCKERFILE, _REQUIREMENTS]
 
 
-def parse_variables(ctx, param, value):
-    """Parse template variables to dictionary."""
-    try:
-        variables = ast.literal_eval(value)
-    except ValueError:
-        raise_template_error(value)
-    if type(variables) is not dict:
-        raise_template_error(value)
-    return variables
-
-
-def raise_template_error(value):
-    """Raise template error with short explanation."""
-    error_info = [
-        '{0}'.format(value), 'Tip: a dictionary is expected',
-        (
-            'Example: --template-variables '
-            '\'{ "variable_1": "string", "variable_2": 2 }\''
-        )
-    ]
-    raise errors.ParameterError(
-        '\n'.join(error_info), '"--template-variables"'
-    )
+def parse_parameters(ctx, param, value):
+    """Parse parameters to dictionary."""
+    parameters = {}
+    for index, parameter in enumerate(value):
+        splitted = parameter.split('=', 1)
+        if len(splitted) < 2 or len(splitted[0]) < 1:
+            raise errors.ParameterError(
+                'Parameter format must be --parameter "param1"="value". ',
+                f'--parameter #{index + 1}'
```

Comment:
It's better to show the parameter itself instead of its index: `f'"{parameter}"`

Reply:
this makes sense :+1: